src/plugin.rs

```diff
@@ -25,6 +25,33 @@ impl ProtoPlugin {
 		Self {
 			options: Some(ProtoDataOptions {
 				directories: vec![dir.to_string()],
+				recursive_loading: false,
+				deserializer: Box::new(DefaultProtoDeserializer),
+				extensions: Some(vec!["yaml", "json"]),
+			}),
+		}
+	}
+
+	/// Same as with_dir but recursively loads prototype files.
```

Comment:
Would be better to link this directly to the method so it's easier to find.

```suggestion
	/// Same as [with_dir] but recursively loads prototype files.
```